include/interrupt.h

```diff
@@ -56,10 +58,33 @@ void set_SI_interrupt( int active );
 void set_SP_interrupt( int active );
 void set_TI_interrupt( int active );
 void set_CART_interrupt( int active );
+void set_RESET_interrupt( int active );
+
+/** 
+ * @brief Guaranteed length of the reset time.
+ * 
+ * This is the guaranteed length of the reset time, that is the time
+ * that goes between the user pressing the reset button, and the CPU actually
+ * resetting. See #exception_reset_time for more details.
+ * 
+ * @note The general knowledge about this is that the reset time should be
+ *       500 ms. Testing on different consoles show that, while most seem to
+ *       reset after 500 ms, a few EU models reset after 200ms. So we define
+ *       the timer shorter for greater compatibility.
+ */
+#define RESET_TIME_LENGTH      TICKS_FROM_MS(200)
+
+uint32_t exception_reset_time( void );
```

Comment:
I think we may need to document this in the wiki as someone might have imported the other header already

Reply:
Actually I think the only public API is including `libdragon.h`. I always considered moving stuff across include files a non-breaking change. We might want to document this better, but all our examples and all user code I have ever seen only includes libdragon.h.